github/github-accessors.go

```diff
@@ -0,0 +1,7229 @@
+// Copyright 2017 The go-github AUTHORS. All rights reserved.
+//
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+// github-accessors.go generated by gen-accessors; DO NOT EDIT
```

Comment:
The [standardized generated file disclaimer header format](https://github.com/golang/go/issues/13560#issuecomment-277804473) not official yet, but it seems to be on its way to become so. Let's start following that standard.

```
// Code generated by gen-accessors; DO NOT EDIT.
```